addons/account/models/account_payment_term.py

```diff
@@ -31,23 +31,33 @@ def _check_lines(self):
             if len(lines) > 1:
                 raise ValidationError(_('A Payment Term should have only one line of type Balance.'))
 
-    def compute(self, value, date_ref=False, currency=None):
+    def compute(self, company_value, foreign_value, date_ref, currency):
+        """Get the distribution of this payment term.
+
+        :param company_value (float): the amount to pay in the company's currency
+        :param foreign_value (float): the amount to pay in the document's currency
+        :param date_ref (datetime.date): the reference date
+        :param currency (Model<res.currency>): the document's currency
+        :return (list<tuple<datetime.date,tuple<float,float>>>): the amount in the company's currency and
```

Comment:
That's not how `:return:` works. It does not take any internal parameter, the type goes into `:rtype:`.

Reply:
Please provide me with guidelines for docstring.
The only thing I can see is in here https://www.odoo.com/documentation/15.0/contributing/development/coding_guidelines.html?highlight=docstring#idiomatics-of-programming-python

> As a good developer, document your code (docstring on methods, simple comments for tricky part of code)

Which I think is achieved here.

If you want to have guidelines on docstring, please discuss it in guru meeting and add a point in the Odoo documentation for that.
There are too many different styles existing in the codebase, and the purpose of documenting is done here IMO.

Reply:
> Please provide me with guidelines for docstring. 

There is no need for "guidelines for docstrings" when it's literally how the tool we're using works: https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html?highlight=domains#info-field-lists